templates/ssr-custom/package.json

```diff
@@ -1,5 +1,8 @@
 {
   "scripts": {
     "start": "NODE_ENV=production node server/index"
+  },
+  "dependencies": {
+    "helmet": "^3.22.0"
```

Comment:
We are keeping package dependencies in template/base/package.json only else it will be difficult to run the app locally